Kitodo-API/src/main/java/org/kitodo/api/dataformat/MediaUnit.java

```diff
@@ -144,6 +171,7 @@ public int hashCode() {
         result = prime * result + ((mediaFiles == null) ? 0 : mediaFiles.hashCode());
```

Comment:
Is it really necessary to build up so complicated hashCode?

Reply:
Writing good hash functions is not easy. The hash function is not only intended to avoid hash collisions as much as possible, but also to distribute as well as possible in order to efficiently allocate hash maps. Because I'm not a professor of theoretical computer science, I prefer to rely on what Eclipse delivers, which is used and developed by computer scientists worldwide. They will do better than me.